samplesheet.wdl

```diff
@@ -0,0 +1,62 @@
+version 1.0
+struct Readgroup {
+    String id
+    File R1
+    File? R2
+}
+
+struct Library {
+    String id
+    Array[Readgroup]+ readgroups
+}
+
+struct Sample {
+    String id
+    Array[Library]+ libraries
+}
+
+task sampleConfigFileToStruct {
```

Comment:
I'm still wondering if it is necessary to have a task for this... If the configs just supply a list of Sample objects, couldn't we just directly read them with read_json (in a scatter for multiple configs) and then flatten it into a single list?

@biowdl/developers I would like to know what everyone thinks about this.

Reply:
This depends on the config format.